approved/v93k/flow_control.tf

```diff
@@ -1655,50 +1655,28 @@ test_flow
   }
   else
   {
-    @T1_BEA7F3B_FAILED = 1;
-  }
-  if @T1_BEA7F3B_FAILED == 1 then
-  {
     run_and_branch(erase1_BEA7F3B)
     then
     {
     }
     else
     {
-      stop_bin "", "fail", , bad, noreprobe, red, 12, over_on;
```

Comment:
The fact that read1 has a later test dependent on its result overrides the bin, so read1 should be a continue on fail instead of binning out to 10.
The 10 should still stick around if the tester API permits it, e.g. in IG-XL it should still appear in the bin column, but on SMT I think the stop_bin statement just disappears.